yt/frontends/ramses/io.py

```diff
@@ -91,28 +91,68 @@ def _read_particle_fields(self, chunks, ptf, selector):
                         data = np.asarray(rv.pop((ptype, field))[mask], "=f8")
                         yield (ptype, field), data
 
+    def _generic_handler(self, fname, foffsets, data_types,
+                         subset, fields):
+        '''General file handler, called by _read_particle_subset
+
+        params:
+        -------
+        fname: filename to read from
+        foffsets: dictionary-like of the offset for the fields
+        data_types: dictionary_like of the data type for the fields
+        subset: a subset object
+        fields: list of fields to read
```

Comment:
We use numpy-style docstrings. See e.g. http://yt-project.org/docs/dev/developing/developing.html#docstrings